prover-sever/src/utils.rs

```diff
@@ -0,0 +1,12 @@
+use log::info;
+use std::fmt::Display;
+
+pub static ERROR_MSG_HEADER: &str = "KROMA";
+
+pub fn kroma_msg<S: AsRef<str> + Display>(msg: S) -> String {
+    format!("[{ERROR_MSG_HEADER}] {msg}")
```

Comment:
is it for error message? then please rename the function name to `kroma_error_msg`.